dev/dll/dllmain.cpp

```diff
@@ -67,7 +67,7 @@ STDAPI DllCanUnloadNow()
 }
 
 _Check_return_
-STDAPI  DllGetClassObject(_In_ REFCLSID rclsid, _In_ REFIID riid, _Outptr_ LPVOID FAR* ppv) 
+STDAPI  DllGetClassObject(REFCLSID rclsid, REFIID riid,  LPVOID FAR* ppv) 
 {
```

Comment:
On classic C-style functions I think we should leave it -- I'd revert this file.